index.html

```diff
@@ -1,12 +1,45 @@
 <!DOCTYPE html>
-  <html lang="en">
-  <head>
-     <meta charset="UTF-8">
-     <title>Angular ToDo App</title>
-     <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
-     <script src="app.js"></script>
-  </head>
-  <body>
-    <h1>To Do!</h1>
-  </body>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Angular ToDo App</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.5.2/angular.min.js"></script>
+  <script src="app.js"></script>
+</head>
+<body data-ng-app="toDoApp">
+  <h1>"Things to Do Today" by Big Black</h1>
+  <main data-ng-controller="itemsController as vm">
+    <table>
+      <thead>
+        <tr>
+          <th>Task Number</th>
+          <th>Task Name</th>
+          <th>Delete</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr data-ng-repeat="item in vm.toDoItems" data-ng-click="toggleEditing(item)">
```

Comment:
Not sure if I'm missing something, but where is `toggleEditing( )` defined? This code seems to be identical to that in your `gh-pages` branch.

Is this something that's native to Angular? New to me!